cloud/amazon/ec2_remote_facts.py

```diff
@@ -83,6 +83,20 @@ def get_instance_info(instance):
     except AttributeError:
         source_dest_check = None
 
+    # Get block device mapping
+    try:
+        bdm_dict = {}
+        bdm = getattr(instance, 'block_device_mapping')
+        for device_name in bdm.keys():
+            bdm_dict[device_name] = {
+                'status': bdm[device_name].status,
+                'volume_id': bdm[device_name].volume_id,
+                'delete_on_termination': bdm[device_name].delete_on_termination,
+                'attach_time': bdm[device_name].attach_time
+            }
+    except AttributeError:
+        instance_info['block_device_mapping'] = {}
```

Comment:
Shouldn't this just be 'pass'?

You've already set bdm_dict to an empty dict and this is then applied to instance_info below